webapi/src/test/scala/org/knora/webapi/responders/admin/ProjectsServiceLiveSpec.scala

```diff
@@ -236,4 +241,26 @@ object ProjectsServiceLiveSpec extends ZIOSpecDefault {
       _ <- projectsService.provide(actorToZioBridge)
     } yield assertTrue(true)
   }
+
+  val getAllProjectDataSpec = test("get all project data") {
```

Comment:
Shouldn't it be rather "get all project**s** data" as the route suggests? Not only this line but in other places across PR you're mixing singular with plural which looks inconsistent. Or I misunderstood it?

Reply:
I think "getAllProjectData" is correct as it is only data from one single project that can be downloaded with this route. I searched for plural froms but couldn't find any. Could you point me to one I've overseen? The route itself starts contains "projects" (and is refined with a certain IRI with ".../Iri/{{projectIdentifier}}")but I don't want to change this as the route should stay the same.

Reply:
I was referring to the plural form in the route itself `/admin/projects` but now after your explanation it makes sense to me too, thx!